### DISCUSSION TITLE
allowed amounts -Average at TIN level

### DISCUSSION THREAD

**Discussion by amerKhan-ctrl** [CMSgov/price-transparency-guide#215] [Q&A] [answered]
Can we report the allowed_amount for a CPT code as the as an average of all the payments made to the provides under the given TIN?

**Comment by ericsilvertx** [comment]
That is a good question! The payment amount does varies from time to time. From a data view point selecting a median value instead of a average might be more meaning full.  Using a Median eliminates or minimize the noise that can occur when you have a few extreme claims with either a very large amount or a very small amount.  I also am calculating the standard deviation when review the data that I am generating

**Comment by brettba** [comment]
Great question! Hopefully we get some answers soon.

**Comment by tdfow** [comment]
The mandate states "Plans and issuers are required to publicly disclose billed charges associated with each unique allowed amount that would be required under the final rules. The final rules further clarify that plans and issuers must report each unique combination of allowed amounts and billed charges for each out-of-network provider, and their associated Place of Service Code, provider NPI, and provider TIN."

**Reply by ericsilvertx** [reply]
Only for the combination where you have 20 or more claims Place of Service Code, provider NPI, and providers TIN "BUT NOT SPECIFIC to a benefit plan."  Can we confirm that we will be looking at only commercials claims that are out of network?

**Comment by shaselton-usds** [comment, marked as answer]
Wrapping up older threads -- no, there are no averaging of the payments. The files required specific billed charges and allowed amounts.
